Build against musl (or current glibc) using the standard name instead of the random implementation detail leakage.

diff -ruN gcc-core/gcc/config/i386/linux-unwind.h gcc-core.bak/gcc/config/i386/linux-unwind.h
--- gcc-core/gcc/config/i386/linux-unwind.h	2006-02-27 11:26:26.000000000 -0600
+++ gcc-core.bak/gcc/config/i386/linux-unwind.h	2014-07-02 18:27:17.931529601 -0500
@@ -137,9 +137,9 @@
     {
       struct rt_sigframe {
 	int sig;
-	struct siginfo *pinfo;
+	siginfo_t *pinfo;
 	void *puc;
-	struct siginfo info;
+	siginfo_t info;
 	struct ucontext uc;
       } *rt_ = context->cfa;
       /* The void * cast is necessary to avoid an aliasing warning.
